virtool/samples/utils.py

```diff
@@ -68,6 +88,16 @@ def get_sample_rights(sample: dict, client):
     return read, write
 
 
+def create_bad_labels_response(labels: List[int]):
+    """
+    Creates a response for a bad request involving labels that do not exist
+
+    :param labels: A list of labels that do not exist in the Labels db
+    :return: A message to give in a `bad_request()` response
+    """
+    return f"Labels do not exist: {', '.join(str(label) for label in labels)}"
+
+
```

Comment:
Add unit tests.

The naming of the function needs to be more accurate. It is not creating a response. That is the duty of `bad_request`. It is creating an error message. You got it right in the `:return:` tag in the docstring.

The wording of the doctring is off. 

```
Creates an error message intended indicating that some label IDs do not exist. labels that do not exist

:param labels: a list of labels to include in the error message
```